crates/sui-node/src/lib.rs

```diff
@@ -309,6 +312,7 @@ impl SuiNode {
         let (p2p_network, discovery_handle, state_sync_handle) = Self::create_p2p_network(
             &config,
             state_sync_store,
+            genesis.checkpoint(),
```

Comment:
minor nit, but if all you need in this function is the ChainIdentifier, can that be what gets passed as an argument instead of passing the entire genesis cp? makes it clearer what it's being used for